kvstore/simplemap/simplemap.go

```diff
@@ -0,0 +1,148 @@
+package simplemap
+
+import (
+	"encoding/hex"
+	"fmt"
+	"io"
+	"slices"
```

Comment:
This requires go 1.21 - might as well update the repo, you'll have to adjust the workflow files in the `.github` directory and update the note in the readme. CI should catch this stuff now

Reply:
Updated to 1.21 :+1: